tests/migrations/test_commands.py

```diff
@@ -13,13 +13,41 @@
 from django.db.backends.base.schema import BaseDatabaseSchemaEditor
 from django.db.migrations.exceptions import InconsistentMigrationHistory
 from django.db.migrations.recorder import MigrationRecorder
-from django.test import override_settings
+from django.test import TestCase, override_settings
 
 from .models import UnicodeModel, UnserializableModel
 from .routers import TestRouter
 from .test_base import MigrationTestBase
 
 
+class AppNameErrorTests(TestCase):
+    def test_migrate_nonexistent_app_error(self):
```

Comment:
I would add a docstring and put this class at the bottom of the file as I think the other test classes will be more commonly used.
```
"""
This class inherits TestCase because MigrationTestBase uses 
`availabe_apps = ['migrations']` which means that it's the only installed
app. 'django.contrib.auth' must be in INSTALLED_APPS for some of these
tests.
"""
```
While you are here, do you want to apply the same behavior to `squashmigrations`?
```
$ python manage.py squashmigrations django.contrib.auth 0005
CommandError: App 'django.contrib.auth' does not have migrations (so squashmigrations on it makes no sense)
$ python manage.py squashmigrations auth 0005
Will squash the following migrations:
 - 0001_initial
 - 0002_alter_permission_name_max_length
 - 0003_alter_user_email_max_length
 - 0004_alter_user_username_opts
 - 0005_alter_user_last_login_null
Do you wish to proceed? [yN] n
````